src/Elasticsearch/Endpoints/Snapshot/Status.php

```diff
@@ -60,7 +60,7 @@ public function setSnapshot($snapshot)
      */
     protected function getURI()
     {
-        if (isset($this->snapshot) === true && isset($this->repository) !== true) {
+        if (isset($this->snapshot) && !isset($this->repository)) {
```

Comment:
Would you mind reverting this change?  The library follows the `foo === true` convention everywhere, and avoids using the inline shorthands (implicit true, `!` negation).